repos/system_upgrade/common/models/custommodifications.py

```diff
@@ -0,0 +1,13 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemFactsTopic
+
+
+class CustomModifications(Model):
+    """Model to store any custom or modified files that are discovered in leapp directories"""
+    topic = SystemFactsTopic
+
+    filename = fields.String()
+    actor_name = fields.String()
+    type = fields.StringEnum(choices=['custom', 'modified'])
```

Comment:
@fernflower I am sorry, I completely overlooked this. the `type` is keyword in python hence we suggest to use different names to not overshadow it. It's not big issue as in this context it has not functional impact, but what about `change_type` or `event_type`?